src/resources/i18n/emailpro/Messages_fr_FR.xml

```diff
@@ -75,7 +75,7 @@
    <translation id="emailpro_tab_domain_remove_success" qtlid="375582">Le domaine va être supprimé de votre service E-mail Pro.</translation>
    <translation id="emailpro_tab_domain_remove_failure" qtlid="375595">Une erreur est survenue lors de la suppression du domaine de votre service E-mail Pro.</translation>
    <translation id="emailpro_ACTION_delegation_step1_intro" qtlid="375608">Une délégation permet à d'autres comptes E-mail Pro d'avoir certains droits sur le compte {0}</translation>
-   <translation id="emailpro_ACTION_add_disclaimer_helpwizard_details" qtlid="375621"><br />Vous pouvez créer une signature par domaine validé.<br /><br />Utilisez les outils à votre disposition pour personnaliser et mettre en forme votre signature.<br /> Cette signature sera intégrée automatiquement au bas de chaque message envoyé depuis les adresses associées à votre domaine E-mail Pro. <br /><br />Des variables sont aussi disponible pour personnaliser la signature générique avec les coordonnées des différents collaborateurs. <br /><br /> Pour ajouter une image ou un logo, utilisez son URL en l'insérant après avoir cliquer sur le bouton "image".</translation>
+   <translation id="emailpro_ACTION_add_disclaimer_helpwizard_details" qtlid="375621"><br />Vous pouvez créer une signature par domaine validé.<br /><br />Utilisez les outils à votre disposition pour personnaliser et mettre en forme votre signature.<br /> Cette signature sera intégrée automatiquement au bas de chaque message envoyé depuis les adresses associées à votre domaine E-mail Pro. <br /><br />Des variables sont aussi disponibles pour personnaliser la signature générique avec les coordonnées des différents collaborateurs. <br /><br /> Pour ajouter une image ou un logo, utilisez son URL en l'insérant après avoir cliquer sur le bouton "image".</translation>
```

Comment:
`cliquer` -> `cliqué`

Reply:
Otherwise, we could not separate the translations with multiple translations to avoid putting an HTML tag in it :p

Reply:
You can use bbcose with the translator for information :
`<translation id="emailpro_ACTION_add_disclaimer_helpwizard_details" qtlid="375621">[br]Vous pouvez créer une signature par domaine validé.
[br]
[br]Utilisez les outils à votre disposition pour personnaliser et mettre en forme votre signature.[br]Cette signature sera intégrée automatiquement au bas de chaque message envoyé depuis les adresses associées à votre domaine E-mail Pro.
[br]
[br]Des variables sont aussi disponibles pour personnaliser la signature générique avec les coordonnées des différents collaborateurs.
[br]
[br]Pour ajouter une image ou un logo, utilisez son URL en l'insérant après avoir cliqué sur le bouton "image".</translation>`

Reply:
No, we shouldn't use bbcode because:

* utils-angular is deprecated ...
* ... and we are switching to angular-translate which don't manage this case
* We should avoid pushing HTML in translations